tests/test_acquisition.py

```diff
@@ -246,6 +308,107 @@ def test_tzinfo(self):
             self.assertTrue(acq.acquisition_datetime, None)
 
 
+class Landsat7MtlRTC2AcquisitionTest(unittest.TestCase):
+    def setUp(self):
+        self.acqs = acquisitions(LS7_SCENERTC2).get_acquisitions(group="RES-GROUP-1")
+
+    def test_type(self):
+        for acq in self.acqs:
+            self.assertTrue(isinstance(acq, LandsatAcquisition))
+
+    def test_band_type(self):
+        self.assertEqual(self.acqs[0].band_type, BandType.REFLECTIVE)
+        self.assertEqual(self.acqs[1].band_type, BandType.REFLECTIVE)
+        self.assertEqual(self.acqs[2].band_type, BandType.REFLECTIVE)
+        self.assertEqual(self.acqs[3].band_type, BandType.REFLECTIVE)
+        self.assertEqual(self.acqs[4].band_type, BandType.REFLECTIVE)
+        self.assertEqual(self.acqs[5].band_type, BandType.THERMAL)
+        self.assertEqual(self.acqs[6].band_type, BandType.THERMAL)
+        self.assertEqual(self.acqs[7].band_type, BandType.REFLECTIVE)
+
+    def test_acquisition_datetime(self):
+        for acq in self.acqs:
+            self.assertEqual(
+                acq.acquisition_datetime,
+                datetime.datetime(2021, 2, 20, 1, 32, 16, 844238),
+            )
+
+    def test_min_radiance_band1(self):
+        self.assertEqual(self.acqs[0].min_radiance, -6.2)
+
+    def test_max_radiance_band1(self):
+        self.assertEqual(self.acqs[0].max_radiance, 191.6)
+
+    def test_min_quantize_band1(self):
+        self.assertEqual(self.acqs[0].min_quantize, 1.0)
+
+    def test_max_quantize_band1(self):
+        self.assertEqual(self.acqs[0].max_quantize, 255.0)
+
+    def test_solar_azimuth(self):
+        self.assertEqual(self.acqs[0].solar_azimuth, 75.86380542)
+
+    def test_solar_elevation(self):
+        self.assertEqual(self.acqs[0].solar_elevation, 42.86386904)
+
+    def test_gain(self):
+        self.assertAlmostEqual(self.acqs[0].gain, 0.77874, 4)
+
+    def test_bias(self):
+        self.assertAlmostEqual(self.acqs[0].bias, -6.97874, 4)
+
+    def test_sensor_id(self):
+        for acq in self.acqs:
+            self.assertEqual(acq.sensor_id, "ETM+")
+
+    def test_platform_id(self):
+        for acq in self.acqs:
+            self.assertEqual(acq.platform_id, "LANDSAT_7")
+
+    def test_samples(self):
```

Comment:
Is this testing the number or count of samples?

Reply:
I'll change the test name.  It is actually a width value

Reply:
Traditionally, the sensors were acquiring `samples` across a given scanline.  Hence the term `samples`.  And the USGS supplied metadata would also use the terms `lines` and `samples`.  I don't mind either way, whatever makes it easier for someone to maintain.